composer.json

```diff
@@ -16,7 +16,7 @@
     "license" : "MIT",
     "require" : {
         "masterminds/html5" : "^2.1.2",
-        "twig/twig" : "^1.10||^2.0",
+        "twig/twig" : "^1.28||^2.0",
```

Comment:
Moreover, the actual Twital version isn't compatible with Twig 1.10. The interface `Twig_ExistsLoaderInterface` was introduced in Twig 1.11.